src/components/program/ProgramNotFound.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from 'react';
```

Comment:
@HammadAhmadWaqas  Can we use/extend the existing NotFoundPage component instead of creating a new one?

Reply:
Good idea, let me make NotFoundPage dynamic